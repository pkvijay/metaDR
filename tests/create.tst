create BADPATH 123
create /test 123
create /test 123
create /test/b abc
create /foo/c def
