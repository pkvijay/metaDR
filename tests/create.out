MALFORMED KEY
CREATED
KEY ALREADY EXISTS
CREATED
NO PARENT
